cwl_utils/graph_split.py

```diff
@@ -177,5 +198,17 @@ def seen_import(entry: MutableMapping[str, Any]) -> bool:
     return seen_imports
 
 
+def json_dump(entry: Any, output_file: str):
```

Comment:
> error: Function is missing a return type annotation

You can run `make mypy` or `tox -e py38-mypy` to check before pushing

Reply:
I'm sorry I didn't notice these, but where can I find a contribution guide?